contrib/epee/src/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ file(GLOB EPEE_HEADERS_PUBLIC
 	"${EPEE_INCLUDE_DIR_BASE}/**/*.h*" # Any number of subdirs will be included.
 )
 
-add_library(epee STATIC byte_slice.cpp byte_stream.cpp hex.cpp abstract_http_client.cpp http_auth.cpp mlog.cpp net_helper.cpp net_utils_base.cpp string_tools.cpp
+add_library(epee byte_slice.cpp byte_stream.cpp hex.cpp abstract_http_client.cpp http_auth.cpp mlog.cpp net_helper.cpp net_utils_base.cpp string_tools.cpp
     wipeable_string.cpp levin_base.cpp memwipe.c connection_basic.cpp network_throttle.cpp network_throttle-detail.cpp mlocker.cpp buffer.cpp net_ssl.cpp
```

Comment:
Already addressed here in a smaller, more managable PR : https://github.com/monero-project/monero/pull/7416

Reply:
Actually it won't hurt to leave it. Let's just push it through. My PR also changes one thing more.